src/tunacode/ui/renderers/tools/base.py

```
@@ -419,6 +444,27 @@ def pad_viewport_lines(self, lines: list[str]) -> list[str]:
             result.append("")
         return result
 
+    def build_panel(
+        self,
+        content: RenderableType,
+        data: T,
+        max_line_width: int,
+    ) -> Panel:
+        timestamp = datetime.now().strftime("%H:%M:%S")
```

Ruff (0.14.14)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)